gandalf.js

```diff
@@ -2,117 +2,110 @@
  * Extras Plugin
```

Comment:
SCRIPT-I'm a bit concerned about the potential performance impact of this loop. Have you thought about optimizing it?